easybuild/easyblocks/c/cp2k.py

```diff
@@ -90,27 +93,28 @@ def extra_options():
         extra_vars = {
             'extracflags': ['', "Extra CFLAGS to be added", CUSTOM],
             'extradflags': ['', "Extra DFLAGS to be added", CUSTOM],
-            'ignore_regtest_fails': [False, ("Ignore failures in regression test "
-                                             "(should be used with care)"), CUSTOM],
+            'ignore_regtest_fails': [False, "Ignore failures in regression test", CUSTOM],
             'library': [False, "Also build CP2K as a library", CUSTOM],
-            'maxtasks': [4, ("Maximum number of CP2K instances run at "
-                             "the same time during testing"), CUSTOM],
-            'modinc': [[], ("List of modinc's to use (*.f90], or 'True' to use "
-                            "all found at given prefix"), CUSTOM],
+            'modinc': [[], ("List of modinc's to use (*.f90), or 'True' to use all found at given prefix"), CUSTOM],
             'modincprefix': ['', "Intel MKL prefix for modinc include dir", CUSTOM],
             'runtest': [True, "Build and run CP2K tests", CUSTOM],
             'omp_num_threads': [None, "Value to set $OMP_NUM_THREADS to during testing", CUSTOM],
             'plumed': [None, "Enable PLUMED support", CUSTOM],
             'type': ['popt', "Type of build ('popt' or 'psmp')", CUSTOM],
             'typeopt': [True, "Enable optimization", CUSTOM],
+            'tests_maxtasks': [4, "--maxtasks in regtest command", CUSTOM],
+            'tests_mpiranks': [1, "--mpiranks in regtest command", CUSTOM],
```

Comment:
This should probably be `2`, to match the default in CP2K's `do_regtest.py` script (and also since with `1` we're seeing test failures with CP2K 2024.x which are not there when using `2` MPI ranks, cfr. https://github.com/easybuilders/easybuild-easyconfigs/pull/21720)